tests/lib/rules/no-classic-classes.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Disallow "classic" classes in favor of native JS classes
+ * @author Alex LaFroscia
+ */
```

Comment:
Let's remove these comments.

Reply:
No problem -- they're just generated by default by the Yeoman generator